selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/AbstractDevice.java

```diff
@@ -607,4 +607,31 @@ public String getModel() {
   public String getAPITargetType() {
     return apiTargetType;
   }
+
+  public void unlockScreen() throws AndroidDeviceException {
+    // Get phone's android version and whether the screen is off.
+    // Different ways to detect if the screen is off depending on the version.
+    CommandLine commandVersion = adbCommand("shell", "getprop", "ro.build.version.release");
```

Comment:
can you use this.getTargetPlatform() instead? And then just check targetPlatform.getVersionNumber() or .getApi() (parseInt and check for > 19 for lollipop)